checker.sh

```diff
@@ -0,0 +1,18 @@
+venv/bin/python3 -m pytype --config backend/pytype.config
```

Comment:
Make is for making

Reply:
I don't see a problem in using bash
This script is usable for checking your code before push (for example I configured git pre-push hook)
Because the same code runs in github actions. Maybe there should be a better way to make this work